src/config.ts

```diff
@@ -244,7 +245,7 @@ export function buildOverriddenAgentConfig(
     artifactsAddressesMap(),
   );
   const filteredAddressesMap: ChainMap<HyperlaneAgentAddresses> = objFilter(
-    mergedAddressesMap,
+    mergedAddressesMap as unknown as ChainMap<HyperlaneAgentAddresses>,
```

Comment:
Would it be more succinct to do the `as` cast to the result of `objFilter` instead of casting here and declaring the type in the line above?